src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,37 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.Arrays;
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String SEPARATOR = "-";
+    private static final int START_PERIOD = 0;
+    private static final int END_PERIOD = 1;
+    private static final int MIN_AGE = 35;
+    private static final String NATIONALITY = "Ukrainian";
+    private static final int MIN_YEARS_IN_UKRAINE = 10;
+
+    public static boolean isEligibleForPresident(Candidate candidate) {
```

Comment:
Let's incapsulate it.
```suggestion
    private static boolean isEligibleForPresident(Candidate candidate) {
```